### PR TITLE
new dir to encapsulate upload command

### PR DESCRIPTION
**Pull request by Scott-Canning**
Created `src/commands` directory to encapsulate commands outside of `cli.js` and moved the `upload` command

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -1,37 +1,11 @@
...
-        throw new Error(
-          'missing private key as base64 of a wallet.json or missing wallet.json',
-        )
-      }
-
-      return await upload(directory, privateKey, dryRun)
-    },
-  )
+  .command(upload.command, upload.description, upload.builder, upload.handler)
   .option('private-key', {
```
Comment (aboutlo) [inline src/cli.js:9]: Please try to move both the options in the `builder` function in the upload command.

File: src/commands/upload.js
Comment (aboutlo) [inline src/commands/upload.js:1]: Let's rename it as `uploadCommand.js` given we already have the `upload.js` file with the actual implementation

**Review by aboutlo — CHANGES_REQUESTED** [review]
Looks good a couple of potential improvement before merging it

**Comment by Scott-Canning** [comment]
Changed `commands/upload.js` to `commands/uploadCommand.js` and moved `upload` options into the associated `builder` function.

Test locally with `npx . upload build/ --private-key ${PRIVATE_KEY} --dry-run`

**Review by aboutlo — APPROVED** [review]
LGTM

**Pull request merged**